Makefile

```diff
@@ -1,9 +1,7 @@
-PIP_HOME = $(shell python3 -c "import site; import os; print(os.path.join(site.USER_BASE, 'bin'))" \
-	|| python -c "import site; import os; print(os.path.join(site.USER_BASE, 'bin'))")
-
-VENV := .venv
-BIN := $(VENV)/bin
-PYTHON := $(BIN)/python
+CONDA_ENV := hknweb
+PYTHON_VERSION := 3.7.3
+# Require Python be ran insie Conda, otherwise directory error out
+PYTHON := "$(CONDA_PREFIX)"/python
```

Comment:
My conda python seems to be at "$(CONDA_PREFIX)"/bin/python

Reply:
Pretty sure when you `conda activate ...` it'll set the env var for python to point to the correct python, but I don't have a link for this. A test is `which python; conda activate ...; which python`.

Reply:
Yes it does
Otherwise, it is blank
"/python" usually doesn't exist unless someone on purpose tried to make one

Following similar design decision and precedence of using venv Python, we try to target the hknweb Conda Python to avoid using another Python by accident

Yes, "which python" does show the Python path, but if not in conda environment, it uses a non-conda Python (unless they check the box to use Conda Python by default)

Reply:
It's possible to write a "make check-conda" and copy paste it on each Makefile command

I will investigate that as well

Reply:
Conda sanity checks pushed via commit ee219db